pycm/cm.py

```diff
@@ -631,6 +631,20 @@ def distance(self, metric):
                 TP=self.TP[i], FP=self.FP[i], FN=self.FN[i], TN=self.TN[i])
         return distance_dict
 
+    def dissimilarity_matrix(self):
+        """
+        Calculate dissimilarity matrix.
+
+        :return: dissimilarity matrix as dict
+        """
+        result = {class_name: dict(zip(self.classes, [0] * len(self.classes))) for class_name in self.classes}
```

Comment:
why not:?
```
result = {c1: {c2: 0} for c2 in self.classes for c1 in self.classes}
```

Reply:
The `zip` function is faster, especially when dealing with large numbers.